packages/sirius-web/backend/sirius-web-sample-application/src/test/java/org/eclipse/sirius/web/sample/tests/integration/task/DeckControllerIntegrationTests.java

```diff
@@ -14,51 +14,57 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.util.Optional;
+import com.jayway.jsonpath.JsonPath;
 
-import org.eclipse.sirius.components.core.api.IRepresentationDescriptionSearchService;
-import org.eclipse.sirius.components.representations.IRepresentationDescription;
-import org.eclipse.sirius.components.task.starter.configuration.view.ViewsStereotypeDescriptionRegistryConfigurer;
-import org.junit.jupiter.api.BeforeEach;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.dto.CreateRepresentationInput;
+import org.eclipse.sirius.components.collaborative.dto.CreateRepresentationSuccessPayload;
+import org.eclipse.sirius.components.graphql.tests.CreateRepresentationMutationRunner;
+import org.eclipse.sirius.web.sample.TestIdentifiers;
+import org.eclipse.sirius.web.sample.tests.integration.AbstractIntegrationTests;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlConfig;
+import org.springframework.transaction.annotation.Transactional;
 
 /**
  * Integration tests of the deck controllers.
  *
  * @author lfasani
  */
 @SuppressWarnings("checkstyle:MultipleStringLiterals")
+@Transactional
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class DeckControllerIntegrationTests extends AbstractTaskControllerIntegrationTests {
-
-    @Autowired
-    private IRepresentationDescriptionSearchService representationDescriptionSearchService;
-
-    @Override
-    @BeforeEach
-    public void beforeEach() {
-        super.beforeEach();
+public class DeckControllerIntegrationTests extends AbstractIntegrationTests {
 
-        this.createStudio(ViewsStereotypeDescriptionRegistryConfigurer.DECK_VIEW_ID);
+    public static final String DECK_DAILY_REPRESENTATION_DESCRIPTION_ID = String.format(
+            "siriusComponents://representationDescription?kind=deckDescription&sourceKind=view&sourceId=%s&sourceElementId=%s", TestIdentifiers.DECK_VIEW, TestIdentifiers.DECK_DAILY_REPRESENTATION);
 
-        this.createTaskProject();
-    }
+    @Autowired
+    private CreateRepresentationMutationRunner createRepresentationMutationRunner;
 
     @Test
-    @DisplayName("Given a task model, deck representation creation succeeds")
+    @DisplayName("Given a task model, when a create representation is performed, then the representation is created")
+    @Sql(scripts = { "/scripts/initialize.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = { "/scripts/cleanup.sql" }, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
     public void givenGanttThenTasksMutationsSucceed() {
```

Comment:
givenGanttThenTasksMutationsSucceed to update to givenTaskModelThenGanttRepresentationCreationSucceeds

Reply:
done